### DISCUSSION TITLE
Unable to make Standalone Mode Work

### DISCUSSION THREAD

**Discussion by sgriffin1717** [noDRM/DeDRM_tools#333] [General]
I have been trying to get the  standalone mode to work. Currently i run the command:

python3 DeDRM_plugin.zip remove_drm X.azw3 -o "Y.azw3"

The following is then printed
Executing 'remove_drm' on file(s) ['X.azw3'] with parameters ['--output', 'Y.azw3']
File /path/to/X.azw3 to /path/to/Y.azw3

But no file is actually printed at that location. Is there a gotcha of some sort that I am missing? I am using the latest released zip file. I also have a "dedrm.json" file in the same directory as the DeDRM_plugin.zip file. 

Any help will be greatly appreciated!

**Comment by noDRM** [comment]
The standalone mode is still in development and it's not working (yet). It was an experiment that I started working on, but it's not finished yet, so for now you need Calibre to use this plugin.
